homeassistant/components/zwave_js/services.py

```diff
@@ -57,6 +57,10 @@
     vol.Optional(ATTR_ENTITY_ID): cv.entity_ids,
 }
 
+API_PLACEHOLDERS = {
+    "api_method": "https://zwave-js.github.io/node-zwave-js/#/api/CCs/index",
+    "api_parameters": "https://zwave-js.github.io/node-zwave-js/#/api/CCs/index",
+    }
```

Comment:
```suggestion
```
Remove this lines, and only add one placeholder for `api_url` (URLs are the same), here:

https://github.com/home-assistant/core/blob/973405822b1e8467c58e99a1c8da8ec6365a4324/homeassistant/components/zwave_js/services.py#L433-L455